did-pkh-method-draft.md

```diff
@@ -0,0 +1,256 @@
+
+There are over hundreds of billions of on-chain, balance-holding accounts across
+the major 50 or so blockchains, all secured and namespaced using similar
+technologies. Almost all of these derive identifiers from hashed or otherwise
+obscured public keys, which are provided at time of transaction.
+
+These accounts are used to protect billions of dollars in assets and critical
+digital infrastructure. They are also actively being piloted and used by
+enterprises and governments. They are rapidly becoming a major form of shared
+data infrastructure across verticals and continents.
+
+DIDs should favor usability where possible, and it is extremely beneficial from
+a security & human computer interaction perspective to have DIDs that readily
+correspond to their equivalents on decentralized networks. This corresponds
+neatly to end-users' understanding of what an "account" is on an existing
+network.  There are knock-on security effects to having an
+immediately-recognizable "address" double as a DID for usage elsewhere.  
+
+It also allows most if not all blockchain accounts to instantly leverage an
+existing identity/account and deploy a W3C Decentralized Identifier from it in a
+standards-conformant way. This "DID-wrapping" of an existing identifier can be
+used in combination with other DID-compatible technologies, such as W3C
+Verifiable Credentials or Authorization Capabilities, and produce proper
+signature-suite definitions, such as "metamask-signing" (signing according to
+the [[eip712]] protocol, soon to be a work item at W3C-CCG).
+
+### Relationship to other DID architectures
+
+did:pkh bears many similarities to
+[did:key](https://w3c-ccg.github.io/did-method-key/#introduction) except it is
+optimized for identifiers derived from hashes of public keys according to
+well-known algorithms (commonly referred to as "public key hashes" because in
+most cases they are a public key hashed according to a standard hash function).
+
+### Combination with other DID methods
+
+Another difference from did:key is that did:pkh is design to have many "upgrade
+paths" for DIDs deterministically generated from existing keypairs.  Namely:
+- if a did:pkh is controlled by a keypair which is valid for generating a
+  blockchain-published DID document according to another method (for instance,
+  did:tz, did:btcr or did:ethr), its did document can be translated to the form
+  of that method's documents, and it can be registered there.
+
+## Design Goals
+
+1. The primary goal of this method is to allow any valid blockchain address to
+   "spin up" a feature-limited but valid and widely interoperable DID and DID
+   Document, valid in a limited context where accounts are represented by DIDs. 
+2. This method is very narrow and unopinionated to allow a wide range of
+   implementations.
+3. For example, the validity of each address to be wrapped in a DID is checked
+   according to the [CAIP-10] standard before generating, to prevent a did:pkh
```

Comment:
Should include a link to the caip-10 spec here.

Reply:
https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-10.md

Reply:
![image](https://user-images.githubusercontent.com/37127325/140987681-af40f175-e2f1-49e1-ad5a-ba0edb4b8326.png)
 
It's handled at the end of the doc in fancy markdown footnotes that don't display in GH unless you view it in code or raw mode 😄

Reply:
More about reference-style links:
- https://github.github.com/gfm/#link-reference-definitions
- https://daringfireball.net/projects/markdown/syntax#link